src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -624,13 +642,16 @@ private function buildTableRows(array $rows): TableRows
 
         return new TableRows(function () use ($rows, $unmergedRows): \Traversable {
             foreach ($rows as $rowKey => $row) {
-                yield $row instanceof TableSeparator ? $row : $this->fillCells($row);
+                $rowGroup = [
+                    $row instanceof TableSeparator ? $row : $this->fillCells($row),
+                ];
```

Comment:
as per our coding standards
```suggestion
        $rowGroup = [$row instanceof TableSeparator ? $row : $this->fillCells($row)];
```